packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md

```diff
@@ -183,6 +186,17 @@ The OIN Wizard journey includes the **Test integration** experience page to help
 
 > **Note:** You must have the Okta Browser Plugin installed with **Allow in Incognito** enabled before you use the  **OIN Submission Tester**. See [OIN Wizard requirements](/docs/guides/submit-app-prereq/main/#oin-wizard-requirements).
 
+#### Navigate directly to test your integration
+
+You can navigate directly to the OIN Wizard **Test integration** page if you have an existing submission in the **Your OIN Integrations** dashboard. This option is only available if your submission has protocol, configuration, and test information already saved. You can bypass the **[Select protocol](#start-a-submission)**, **[Configure your integration](#configure-your-integration)**, and **[Test your integration](#enter-test-information)** pages in the OIN Wizard, and start generating instances for testing. This saves you time and avoids unnecessary updates to an existing integration submission.
+
+To navigate directly to test your integration:
```

Comment:
```suggestion
Follow these steps to test your integration:
```

Reply:
This is a shortcut to "test your integration". Maybe I need to rework this part. Thanks